pango-view/viewer-pangocairo.c

```diff
@@ -272,7 +273,11 @@ render_callback (PangoLayout *layout,
       cairo_restore (cr);
     }
 
-  cairo_move_to (cr, 0, 0);
+  if (opt_trim)
+    cairo_move_to (cr, opt_trim_x, opt_trim_y);
+  else
+    cairo_move_to (cr, 0, 0);
+
```

Comment:
This if condition is superfluous, if `opt_trim` is not set, `opt_trim_x` and `opt_trim_y` will be zero anyway.

Reply:
Fixed, after seeding the new struct LayoutExtents with zero values.